api/src/main/java/org/pmiops/workbench/actionaudit/auditors/EgressEventAuditorImpl.kt

```diff
@@ -36,12 +36,21 @@ constructor(
      * flow logs. Empirically, an "-m" suffix is added to the VM name, due to Leo team's use
      * of Dataproc (see https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/metadata).
      *
-     * TODO(RW-4848): Update this to account for GCE naming conventions.
+     * TODO(RW-4848): How do egress alerts interact with worker nodes?
```

Comment:
Also while you're here: any initial thoughts on this?

My assumptions are:
- The existing egress alerting already covers traffic coming from Dataproc workers nodes.
- The naming convention for these nodes is not handled anywhere, so we may need to do some more clever mapping here to determine the owner.
- Egress from worker nodes is potentially a big red flag, and if it were easy, I might be in favor of locking it down. There is no real reason to allow worker nodes to send a lot of data out of the cluster

Reply:
All good points – nothing I'd add on top of that.